tests/models/speech_to_text/test_modeling_speech_to_text.py

```diff
@@ -759,10 +759,6 @@ def test_pt_tf_model_equivalence(self, allow_missing_keys=True):
         # Allow missing keys since TF doesn't cache the sinusoidal embeddings in an attribute
         super().test_pt_tf_model_equivalence(allow_missing_keys=allow_missing_keys)
 
-    @unittest.skip("Test failing,  @RocketNight is looking into it")
```

Comment:
Actually, we need to keep this skip, it is what is causing the failure in CI!

Reply:
Done.